### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by Chike1990**
In this milestone, I built the project schema;  create and run the necessary migration files with table and column names as shown in the ERD diagram

### CODE REVIEW

File: db/migrate/20220811142452_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users } 
+      t.references :post, null: false, foreign_key: true
+
+      t.timestamps
```
Comment (Kingstalux) [inline db/migrate/20220811142452_create_comments.rb:7]: - [ ] Good job so far :+1: A comment should also have a `text` according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Kindly address this issue :+1:

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Chike1990 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :clap: 
:ballot_box_with_check: Good job following the ERD diagram :fire: 
:ballot_box_with_check: Linters are passing :ok_hand: 
:ballot_box_with_check: Good commit history :clap: 


## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly change the name of your `main` branch to `dev` so as to follow the correct Gitflow :100: 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220811142452_create_comments.rb (hunk 1 above)
Comment (Chike1990) [reply db/migrate/20220811142452_create_comments.rb]: Alright, thank you so much

**Review by mikethreels — APPROVED** [review]
Hi Chibuike, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chike1990** [comment]
Thank you!!

**Pull request merged**